src/Adapters/Files/FilesAdapter.js

```diff
@@ -56,6 +59,37 @@ export class FilesAdapter {
    * @return {string} Absolute URL
    */
   getFileLocation(config: Config, filename: string): string {}
+
+  /** Validate a filename for this adaptor type (optional)1G
```

Comment:
1G? I also see you are using adaptor instead of adapter. Not to nitpick but can you change it for consistency.

Reply:
How embarrassing